collector/src/compare.rs

```diff
@@ -10,11 +10,29 @@ use tabled::{Table, Tabled};
 /// Compare 2 artifacts and print the result.
 pub async fn compare_artifacts(
     mut conn: Box<dyn Connection>,
-    base: String,
-    modified: String,
+    base: Option<String>,
+    modified: Option<String>,
 ) -> anyhow::Result<()> {
     let index = database::Index::load(&mut *conn).await;
 
+    let aids = index.artifacts().collect::<Vec<_>>();
+    if aids.len() < 2 {
+        return Err(anyhow::anyhow!("There's no artifacts to compare."));
```

Comment:
This can be misleading if there's only a single artifact. Let's say "There are not enough artifacts to compare, at least two are needded".